engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/desugar/GenerateMethodBodies.scala

```diff
@@ -161,7 +160,7 @@ case object GenerateMethodBodies extends IRPass {
           case lam @ Function.Lambda(_, body, _, _, _, _)
               if findForeignDefinition(
                 body,
-                lang = Some(ForeignLanguage.JS)
+                lang = Some("js")
```

Comment:
Can we keep that as a constant somewhere, please?

Reply:
Too much overhead to do so. We would need a new project sharable between all that were using `ForeginLanguage.JS`. But I want to get rid of the heavyweight `polyglot-api` dependency (in `EpbLanguage` at least). I don't want to create new project just to host `js` constant.

Btw. the real constant is defined in `graal-js` and we don't compile against such module. E.g. there was never any guarantee (except tests) that we are using the right constant.